src/commands/misc/httpdog.js

```diff
@@ -0,0 +1,21 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+module.exports = class HttpDog extends Command {
+  constructor (client) {
+    super({
+      name: 'httpdog',
+      category: 'general',
+      parameters: [{
+        type: 'number',
+        required: false

```

Comment:
o

Reply:
It has a default value of `200`, hence it's not required. Also this PR has been closed already

Reply:
i just sent a fucking o